examples/experimental/dagster-dlift/dagster_dlift/translator.py

```diff
@@ -29,6 +30,15 @@ class DbtCloudContentData:
     content_type: DbtCloudContentType
     properties: Mapping[str, Any]
 
+    def from_raw_gql(self, raw_data: Mapping[str, Any]) -> "DbtCloudContentData":
+        content_type = DbtCloudContentType(raw_data["resourceType"].upper())
+        return DbtCloudContentData(content_type=content_type, properties=raw_data["properties"])
+
+
+def clean_asset_name(name: str) -> str:
```

Comment:
we really ought to stick this in the framework somewhere

Reply:
:nod2:

Reply:
I ended up adding it to `dagster._utils.names`